### DISCUSSION TITLE
Request: shortcut customization for w10

### DISCUSSION THREAD

**Discussion by LeDuble** [flameshot-org/flameshot#2546] [Ideas]
Can we have shortcut customization for "capture screen". Seems like its unavailable to be modified. I find prnt screen button is not intuitive for me.

In linux it was easier to customize this shortcut for this button but I haven't been able to find solution in windows for it.

Thanks in advance!

**Comment by mmahmoudian** [comment]
This is practically a duplicate of #1341

If you are stuck with Windows, you can use [this workaround](https://github.com/flameshot-org/flameshot/issues/1861#issuecomment-910552032)
